openage/util/fslike/path.py

```diff
@@ -32,7 +33,7 @@ class Path:
     # lower.
     # pylint: disable=too-many-public-methods
 
-    def __init__(self, fsobj, parts=None):
+    def __init__(self, fsobj, parts=None, is_temp=False):
```

Comment:
This parameter should not be set here since it implies that you could create the Path via `__init__` as a tempfile. This is not what we want (people should use the static method you implemented)